docusaurus/docs/gers/gers.mdx

```diff
@@ -5,26 +5,24 @@ label: GERS
 
 Overture's Global Entity Reference System (GERS) is a framework for structuring, encoding and referencing map data to a shared universal reference. Overture is building this common ID system to solve the difficult problem of integrating geospatial datasets; to make it easier to share and sell map data; and to enable cooperation among companies, non-profits, government agencies and other organizations.
 
-## Defining an entity
+GERS provides a mechanism to conflate datasets, matching one or more features via a unique GERS ID. For example, two geospatial datasets each containing a polygon that represents the footprint of the Empire State Building can be easily conflated because both polygons will contain the same GERS ID. The resulting dataset will have a single polygon feature, with one GERS ID, that represents the real-world entity known as: "New York's Empire State Building."
 
-GERS provides a mechanism to conflate datasets, matching _entity_ to _entity_ via a unique GERS ID. An _entity_ is basically a thing in the real world. That thing could be a segment of road, a city, a grocery store, a building, a park bench or a lamp post. An _entity_ in GERS is assigned a unique GERS ID. These IDs are useful to anyone looking to match their data with Overture data.
+## What exactly is an entity?
+Overture defines an entity as a physical thing or concept in the real world, represented as a feature in a geospatial dataset. It could be a segment of road, a city boundary, a grocery store, a building, a park bench or a lamp post. Overture assigns unique IDs, called GERS IDs, to these features. In most cases it's helpful to think of an entity and a feature as the same thing, but in practice it can be more complicated. An entity in the world could be represented by multiple features in a dataset, and a feature in a dataset might be a representation of multiple entities in the world. For example, a school building and its entrances and exits might be considered a single entity in the world but could be represented as multple features in a dataset, each feature with its own GERS ID.
```

Comment:
i know this was already there and yes, could be indeed, but i wonder if this may be confusing new users that we have these in our data set, since they exist in osm for example;
i would stick with examples from types of data we already have, afaik benches and lamps are not in overture data set and might not be there anytime soon or at all.